### PR TITLE
Update Active Job testing guide

### PR DESCRIPTION
**Pull request by ghiculescu**
The testing guide for Active Job currently implies that when you queue a job it will be performed:

image

This isn't true; by default jobs are enqueued, not performed.

This PR fleshes out the docs a bit to show both examples, and adds a test to confirm the default behaviour.

### CODE REVIEW

**Review by zzak — APPROVED** [review]

**Comment by ghiculescu** [comment]
@zzak guessing you can't merge this because of the test?

**Comment by zzak** [comment]
@ghiculescu No, I generally let the author merge once they have commit to merge their own PR. 🙏

**Comment by ghiculescu** [comment]
I don’t have commit @zzak

**Comment by zzak** [comment]
Whoops, I thought I've seen you merge PRs before! I will merge this then :pray:

**Pull request merged**
